examples/RaceConditions.cpp

```diff
@@ -0,0 +1,37 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2024, Lawrence Livermore National Security, LLC and CARE
+// project contributors. See the CARE LICENSE file for details.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//////////////////////////////////////////////////////////////////////////////
+
+// CARE library header
+#include "care/care.h"
+#include "care/algorithm.h"
+
+// Std library headers
+#include <algorithm>
+#include <iostream>
+#include <limits>
+#include <vector>
+
+int main(int, char**) {
+   int size = 10;
```

Comment:
this is nitpicky, but you could have size as a constant

Reply:
Fixed.